README.md

```diff
@@ -31,6 +31,8 @@ npm start
 - [Hybrids](https://github.com/webcomponents/custom-elements-everywhere/tree/master/libraries/hybrids)
 - [Hyperapp](https://github.com/webcomponents/custom-elements-everywhere/tree/master/libraries/hyperapp)
 - [HyperHTML](https://github.com/webcomponents/custom-elements-everywhere/tree/master/libraries/hyperhtml)
+- [Lit](https://github.com/webcomponents/custom-elements-everywhere/tree/master/libraries/lit)
```

Comment:
It's nice that we're adding this – is this the only existing component that was missed here?

Reply:
Looks like `React@experimental` was also missed

Reply:
Nice catch. Thanks!